models/cart-discount-value-gift-line-item/src/cart-discount-value-gift-line-item-draft/presets/empty.ts

```diff
@@ -0,0 +1,9 @@
+import type { TCartDiscountValueGiftLineItemDraftBuilder } from '../../types';
+import CartDiscountValueGiftLineItemBuilder from '../builder';
+
+const empty = (): TCartDiscountValueGiftLineItemDraftBuilder =>
+  CartDiscountValueGiftLineItemBuilder()
+    .distributionChannel(undefined)
+    .supplyChannel(undefined);
```

Comment:
The gift line item generator also spoofs id's for distribution and supply channels, so we needed to be able to empty these in order to snapshot test reliably